odoo/addons/base/models/ir_rule.py

```diff
@@ -219,24 +222,21 @@ def get_record_description(rec):
             # If the user has access to the company of the record, add this
             # information in the description to help them to change company
             if company_related and 'company_id' in rec and rec.company_id in self.env.user.company_ids:
-                return f'{rec.display_name} (id={rec.id}, company={rec.company_id.display_name})'
-            return f'{rec.display_name} (id={rec.id})'
-
-        records_description = ', '.join(get_record_description(rec) for rec in records_sudo)
-        failing_records = _("Records: %s", records_description)
+                return f'{description}, {rec.display_name} ({model}: {rec.id}, company={rec.company_id.display_name})'
+            return f'{description}, {rec.display_name} ({model}: {rec.id})'
 
-        user_description = f'{self.env.user.name} (id={self.env.user.id})'
-        failing_user = _("User: %s", user_description)
+        failing_records = '\n '.join(f'- {get_record_description(rec)}' for rec in records_sudo)
 
         rules_description = '\n'.join(f'- {rule.name}' for rule in rules)
-        failing_rules = _("This restriction is due to the following rules:\n%s", rules_description)
+        failing_rules = _("Blame the following rules:\n%s", rules_description)
```

Comment:
Don't like the "blame", like it is an issue to remove. Rules are there for reasons...

Reply:
:arrow_up:  @jija2

Reply:
This is only in debug anyways... Why do we make ***debug*** funny?